plugins/mcreator-core/variables/itemstack.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "itemstack",
+	"color": "0x996069",

```

Comment:
I just thought that maybe we could just use an integer here similar to blockly (hue only), as all colors use blockly lightness and saturation so we only truly need hue here. This also would allow matching colors with procedure blocks more easily as they only specify hue too.